doc/release-notes.md

```diff
@@ -36,10 +36,13 @@ earlier.
 Notable changes
 ===============
 
-Example item
----------------------------------------
+Miscellaneous
```

Comment:
Nit: usually we call this category "RPC low-level API changes"
https://github.com/bitcoin/bitcoin/blob/v0.12.0/doc/release-notes.md#rpc-low-level-api-changes